ValheimPlus/Configurations/BaseConfig.cs

```diff
@@ -93,6 +93,12 @@ public void LoadIniData(KeyDataCollection data)
                     continue;
                 }
 
+                if (prop.PropertyType == typeof(string))
```

Comment:
This currently clashes with the KeyCode property fetch, we are doing things bts to get this prop check to work

Reply:
Not super happy with the configuration approach parsing of all these parameters, having arrays would be a nice addition, let's wait for your next milestone on how you improve your config systems.